ydb/core/sys_view/common/schema.h

```diff
@@ -210,6 +210,7 @@ struct Schema : NIceDb::Schema {
         struct AvailableSize : Column<10, NScheme::NTypeIds::Uint64> {};
         struct TotalSize : Column<11, NScheme::NTypeIds::Uint64> {};
         struct Status : Column<12, NScheme::NTypeIds::Utf8> {};
+        struct State : Column<13, NScheme::NTypeIds::Utf8> {};
```

Comment:
it's better not to reuse 13 -- there is a commented line down here showing this value was reserved